internal/pkg/node/datastructure.go

```diff
@@ -36,7 +36,7 @@ type NodeConf struct {
 	Discoverable   string              `yaml:"discoverable,omitempty"`
 	Profiles       []string            `yaml:"profiles,omitempty"`
 	NetDevs        map[string]*NetDevs `yaml:"network devices,omitempty"`
-	Keys           map[string]string   `yaml:"keys,omitempty"`
+	Tags           map[string]string   `yaml:"tags,omitempty"`
```

Comment:
Be aware of the yaml change here.

Reply:
I don't know of anyone using this (aside from the one case that was using it accidentally thinking it was the assetkey). I can add a bit of code though to automatically convert it and provide backwards compatibility.

Reply:
Done.